libafl_libfuzzer/libafl_libfuzzer_runtime/src/merge.rs

```diff
@@ -236,7 +236,8 @@ pub fn merge(
                     .scheduler_mut()
                     .on_remove(&mut state, idx, &Some(testcase))?;
             } else {
-                #[allow(clippy::needless_borrows_for_generic_args)] // False-positive: file_path is used just below
+                #[allow(clippy::needless_borrows_for_generic_args)]
+                // False-positive: file_path is used just below
```

Comment:
If anything, the comment should be above the allow

Reply:
it was autoformated, i'll change this